### PR TITLE
Data layer integration

### PR DESCRIPTION
**Pull request by tereshkevich1**

### CODE REVIEW

File: app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsBottomSheet.kt [hunk 1]
```diff
@@ -7,7 +7,8 @@ import androidx.compose.material3.rememberModalBottomSheetState
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.res.colorResource
 import com.example.banking.R
-import com.example.banking.presentation.models.Account
+import com.example.banking.data.data_source.Account
+
```
Comment (MaryFilipovichWork) [inline app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsBottomSheet.kt:11]: remove extra empty lines

File: app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsList.kt [hunk 2]
```diff
@@ -18,8 +18,9 @@ import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import com.example.banking.R
+import com.example.banking.data.data_source.Account
 import com.example.banking.presentation.accounts_screen.cards.account.CardAccount
-import com.example.banking.presentation.models.Account
+
```
Comment (MaryFilipovichWork) [inline app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsList.kt:23]: remove extra empty lines

File: app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/GetCardBackgroundColor.kt [hunk 3]
```diff
@@ -7,7 +7,8 @@ import androidx.compose.runtime.getValue
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.colorResource
 import com.example.banking.R
-import com.example.banking.presentation.models.Account
+import com.example.banking.data.data_source.Account
+
```
Comment (MaryFilipovichWork) [inline app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/GetCardBackgroundColor.kt:11]: remove extra empty lines

**Review by MaryFilipovichWork — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsBottomSheet.kt (hunk 1 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsBottomSheet.kt:11]: fixed

File: app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsList.kt (hunk 2 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/AccountsList.kt:23]: fixed

File: app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/GetCardBackgroundColor.kt (hunk 3 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/presentation/accounts_screen/bottomSheet/GetCardBackgroundColor.kt:11]: fixed

**Pull request merged**